File: 백준/Bronze/2480. 주사위 세개/주사위 세개.cc
#include <iostream>

using namespace std;

int main(void) {
    int n1, n2, n3, result = 0;
    
    cin >> n1 >> n2 >> n3;
    
    if (n1 > n2 && n1 > n3) {
        result = n1 * 100;
    } if (n2 > n1 && n2 > n3) {
        result = n2 * 100;
    } if (n3 > n1 && n3 > n2) {
        result = n3 * 100;
    } if (n1 == n2 || n1 == n3) {
        result = 1000 + n1 * 100;
    } if (n2 == n1 || n2 == n3) {
        result = 1000 + n2 * 100;
    } if (n3 == n1 || n3 == n2) {
        result = 1000 + n3 * 100;
    } if (n1 == n2 && n1 == n3) {
        result = 10000 + n1 * 1000;
    }
    
    cout << result << endl;
    return 0;
}

File: 백준/Bronze/2480. 주사위 세개/주사위 세개.py
a, b, c = map(int, input().split())

result = 0
if (a == b == c) :
    result = 10000 + a * 1000
elif (a == b or b == c) :
    result = 1000 + b * 100
elif (b == a or a == c) :
    result = 1000 + a * 100
elif (a == c or c == b) :
    result = 1000 + c * 100
else :
    result = max(a, b, c) * 100

print(result)